index.d.ts

```diff
@@ -4,10 +4,10 @@ import { DefineComponent } from 'vue'
 interface FontAwesomeIconProps {
   border?: boolean
   fixedWidth?: boolean
-  flip?: 'horizontal' | 'vertical' | 'both'
+  flip?: 'horizontal' | 'vertical' | 'both' | boolean
```

Comment:
Hmm. I don't think this is correct. I think the OP was thinking this was the `flip` animation and not the `flip-horizontal`, `flip-vertical`, and `flip-both` properties. @jasonlundien this is confusing because we have "flipping" as a transform and animation operation.

Reply:
we discussed in slack.